evm/src/executor/inspector/cheatcodes/env.rs

```diff
@@ -26,6 +38,27 @@ pub struct Prank {
     pub single_call: bool,
 }
 
+fn broadcast(
+    state: &mut Cheatcodes,
+    origin: Address,
+    original_caller: Address,
+    depth: u64,
+    single_call: bool,
+) -> Result<Bytes, Bytes> {
+    let broadcast = Broadcast { origin, original_caller, depth, single_call };
+
+    if state.prank.is_some() {
+        return Err("You have an active prank. Broadcasting and pranks are not compatible. Disable one or the other".to_string().encode().into());
+    }
```

Comment:
just a suggestion, but looks like a custom `bail!` macro that does this might be useful?

Reply:
don't think so, he's actually expecting `Bytes` here not `Str`